### DISCUSSION TITLE
How in filamentphp 3 app to use custom icons from https://v1.heroicons.com?

### DISCUSSION THREAD

**Discussion by sergeynilov** [filamentphp/filament#9513] [Help] [answered]
In laravel 10 / filamentphp 3 app I to use custom  icons and looking at site  https://v1.heroicons.com/

I made search by “status” and selected icon status-online, which I rendered 'heroicon-o-status-online', loojking at some sample icons like 'heroicon-o-rectangle-stack'
But I got error :

  `  Svg by name "status-online" from set "heroicons" not found`.

Is 'heroicon-o-rectangle-stack' some internal filamentphp key ?
How can I add custom icons from https://v1.heroicons.com/ to my app ?

**Comment by leandrocfe** [comment, marked as answer]
Would you like to use `heroicon-o-signal`?
https://heroicons.com/

**Comment by markosole** [comment]
There's great package that helps with custom SVG icons which takes less then 2 minutes to setup and use. 
More about it here: https://filamentphp.com/docs/3.x/support/icons  and package itself https://github.com/blade-ui-kit/blade-icons
